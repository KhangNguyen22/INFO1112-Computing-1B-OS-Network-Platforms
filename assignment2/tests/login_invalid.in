connect bitbash.xyz 9050
login  bob thebuilder 
quit