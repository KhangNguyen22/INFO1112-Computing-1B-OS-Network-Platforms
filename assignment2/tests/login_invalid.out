> Connected, please log in
> Invalid login details
> 